src/domain/event/__tests__/CreateEventPage.test.tsx

```diff
@@ -0,0 +1,406 @@
+import Router from 'react-router';
+
+import addEventResponse from '../__mocks__/addEventResponse.json';
+import imageMutationResponse from '../__mocks__/imageMutationResponse.json';
+import imageResponse from '../__mocks__/imageResponse.json';
+import keywordResponse from '../__mocks__/keywordResponse.json';
+import keywordsResponse from '../__mocks__/keywordsResponse.json';
+import placeResponse from '../__mocks__/placeResponse.json';
+import placesResponse from '../__mocks__/placesResponse.json';
+import profileResponse from '../__mocks__/profileResponse.json';
+import venueResponse from '../__mocks__/venueResponse.json';
+import { AUTOSUGGEST_OPTIONS_AMOUNT } from '../../../common/components/autoSuggest/contants';
+import {
+  CreateEventDocument,
+  ImageDocument,
+  KeywordsDocument,
+  MyProfileDocument,
+  PlaceDocument,
+  PlacesDocument,
+  UploadSingleImageDocument,
+} from '../../../generated/graphql';
+import apolloClient from '../../app/apollo/apolloClient';
+import { store } from '../../app/store';
+import CreateEventPage from '../CreateEventPage';
+
+expect.extend(toHaveNoViolations);
+
+beforeEach(() => {
+  jest.spyOn(Router, 'useHistory').mockReturnValue({});
+  jest
+    .spyOn(Router, 'useLocation')
+    .mockReturnValue({ pathname: '/', search: '', state: '', hash: '' });
+});
+
+const eventFormData = {
+  name: 'Testitapahtuma',
+  shortDescription: 'Testikuvaus',
+  description: 'Pidempi kuvaus',
+  infoUrl: 'https://www.palvelutarjotin.fi',
+  contactEmail: 'testi@testi.fi',
+  contactPhoneNumber: '123123123',
+  duration: '10',
+  enrolmentStart: '13.08.2020 03:45',
+  enrolmentEndDays: '3',
+  neededOccurrences: '3',
+  imagePhotographerName: 'Valo Valokuvaaja',
+  imageAltText: 'Vaihtoehtoinen kuvateksti',
+};
+
+const createEventVariables = {
+  event: {
+    name: { fi: 'Testitapahtuma' },
+    startTime: '2020-08-07T21:00:00.000Z',
+    offers: [{ isFree: true }],
+    shortDescription: { fi: 'Testikuvaus' },
+    description: { fi: 'Pidempi kuvaus' },
+    images: [{ internalId: '/image/48584/' }],
+    infoUrl: { fi: 'https://www.palvelutarjotin.fi' },
+    audience: [],
+    inLanguage: [
+      { internalId: '/language/en/' },
+      { internalId: '/language/fi/' },
+    ],
+    keywords: [{ internalId: '/keyword/yso:p4363/' }],
+    location: { internalId: '/place/tprek:15417/' },
+    pEvent: {
+      contactEmail: 'testi123@testi123.fi',
+      contactPersonId:
+        'UGVyc29uTm9kZTo0MGZmYTIwMS1mOWJhLTQyZTYtYjY3Ny01MWQyM2Q4OGQ4ZDk=',
+      contactPhoneNumber: '123321123',
+      duration: 10,
+      enrolmentEndDays: 3,
+      enrolmentStart: '2020-08-13T00:45:00.000Z',
+      neededOccurrences: 3,
+    },
+    organisationId: 'T3JnYW5pc2F0aW9uTm9kZToy',
+    draft: false,
+  },
+};
+
+const imageFile = new File(['(⌐□_□)'], 'palvelutarjotin.png', {
+  type: 'image/png',
+});
+
+const mocks = [
+  {
+    request: {
+      query: MyProfileDocument,
+    },
+    result: {
+      ...profileResponse,
+    },
```

Comment:
Maybe this could be:
`result: profileResponse,`

Reply:
yeah